.github/scripts/backport.py

```diff
@@ -0,0 +1,461 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# 	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Backport CLI (label + remind)
+
+Label merged PRs lacking a version label with 'backport pending' or send reminder
+comments on such PRs until a version label (e.g. v9.2) is added.
+
+Quick usage:
+	backport.py help
```

Comment:
`help` alone is not helpful because `--trigger` is required

```
% python .github/scripts/backport.py help                   
usage: backport.py [-h] [--repo REPO] --trigger {pull_request_target,schedule,workflow_dispatch} [--dry-run] [--debug] {label,remind,help} ...
backport.py: error: the following arguments are required: --trigger
```